zds/utils/templatetags/nb_word_to_time.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+
+from django import template
+from zds.settings import ZDS_APP
+register = template.Library()
+
+
+@register.filter(name="nb_word_to_icontime")
+def nb_word_to_icontime(nb_word):
+    """
+    Gets the icon for this time to read the content
+    :param nb_word:
+    :return:
+    """
+    association = ["icon-15-min", "icon-30-min", "icon-45-min", "icon-60-min", "icon-more-min"]
```

Comment:
j'ai tendans à dire que ça permet une page plus belle et accueillante.


Le 22/06/2016 à 20:22, SpaceFox a écrit :
>
> In zds/utils/templatetags/nb_word_to_time.py 
> <https://github.com/zestedesavoir/zds-site/pull/3681#discussion_r68107226>:
>
> > @@ -0,0 +1,25 @@
> > +# coding: utf-8
> > +
> > +from django import template
> > +from zds.settings import ZDS_APP
> > +register = template.Library()
> > +
> > +
> > +@register.filter(name="nb_word_to_icontime")
> > +def nb_word_to_icontime(nb_word):
> > +    """
> > +    Gets the icon for this time to read the content
> > +    :param nb_word:
> > +    :return:
> > +    """
> > +    association = ["icon-15-min", "icon-30-min", "icon-45-min", "icon-60-min", "icon-more-min"]
>
> Ah, et honnêtement je ne vois pas l'intérêt d'une icône, mais on peut 
> en discuter.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3681/files/13758fdcf27f091fc54b1ffb111309ee52aeef78#r68107226>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABc_xRECggawh0UeJQV0ydaCFja2qxIhks5qOX1LgaJpZM4I7sFO>.
>